conf/test_bgc.config

```diff
@@ -16,7 +16,7 @@ params {
 
     // Limit resources so that this can run on GitHub Actions
     max_cpus   = 2
-    max_memory = '6.GB'
+    max_memory = '9.GB'
```

Comment:
I'm a little wary of this, `test` profiles are also often used for running on a laptop. Would `8` be sufficent? That is a typical amount of RAM for less-powerful laptops (following: 4, 8, 16 scheme).

Reply:
according to the run report it needs 7.6 GB, so ill test with 8GB and see if that works

Reply:
i set it to 8GB and it works